templates/partials/header.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="This is an example description">
```

Comment:
Can u remove this default description or make it configurable?

Reply:
I've make it configurable.Now just need to change the line in the `config.toml`
```
description = "This is an example description"
```

Reply:
Please review this PR,as 2 months past,but no further process.Thanks!